console/ladybug/pom.xml

```diff
@@ -0,0 +1,127 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.frankframework</groupId>
+		<artifactId>frankframework-console</artifactId>
+		<version>${revision}</version>
+	</parent>
+
+	<artifactId>frankframework-console-ladybug</artifactId>
+	<name>Frank Console - Ladybug TestTool</name>
+
+	<properties>
+		<ladybug.version>3.0-20240806.151958</ladybug.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.ibissource</groupId>
+			<artifactId>ibis-ladybug</artifactId>
+			<version>${ladybug.version}</version>
+			<exclusions>
+				<exclusion>
+					<groupId>org.codehaus.woodstox</groupId>
+					<artifactId>woodstox-core-asl</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<groupId>org.frankframework</groupId>
+			<artifactId>frankframework-security</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.frankframework</groupId>
+			<artifactId>frankframework-management-gateway</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+		</dependency>
+
+		<!-- Because https://github.com/apache/commons-dbcp has not upgraded to Jakarta yet, we're using the shaded version from Tomcat 10.1 -->
+		<dependency>
+			<groupId>org.apache.tomcat</groupId>
+			<artifactId>tomcat-dbcp</artifactId>
+			<version>${tomcat.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.liquibase</groupId>
+			<artifactId>liquibase-core</artifactId>
+			<version>4.29.2</version>
+			<exclusions>
+				<exclusion>
+					<groupId>javax.xml.bind</groupId>
+					<artifactId>jaxb-api</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+
+		<dependency>
```

Comment:
Does the Ladybug project always need all drivers?
Why not switched with a profile, like we do for the Frank!Framework itself?

Reply:
If it's an executable (Spring Boot) jar you cannot add entries to the classpath. It's therefor required to add all dependencies... 😢

Reply:
Wordt de ladybug-console dan alleen geinclude als je bouwt als Spring Boot app?

Reply:
Jup!